src/components/Layout/TooltipContent.js

```diff
@@ -85,7 +85,10 @@ export const TooltipContent = ({
         return itemDisplayNames
     }
 
-    const renderItems = (itemDisplayNames) => {
+    const renderItems = (itemDisplayNames = []) => {
+        if (itemDisplayNames.findIndex((name) => name === undefined) !== -1) {
```

Comment:
```suggestion
        if (!itemDisplayNames.find((name) => name === undefined)) {
```

Reply:
A bit hard to follow what you're trying to do here though... do you want to make sure that all items to have a `name`?

Reply:
if so then I think this is more clear
```suggestion
        if (itemDisplayNames.some((name) => name === undefined)) {
```

Reply:
or even 
```suggestion
        if (itemDisplayNames.some((name) => !name)) {
```

Reply:
Yeah that looks better